mm2src/mm2_bitcoin/rpc/src/v1/types/mod.rs

```diff
@@ -20,8 +20,9 @@ pub use self::hash::{H160, H256, H264};
 pub use self::script::ScriptType;
 pub use self::transaction::{GetRawTransactionResponse, RawTransaction, SignedTransactionInput,
                             SignedTransactionOutput, Transaction, TransactionInput, TransactionInputEnum,
-                            TransactionInputScript, TransactionOutput, TransactionOutputScript,
-                            TransactionOutputWithAddress, TransactionOutputWithScriptData, TransactionOutputs};
+                            TransactionInputScript, /*TransactionOutput,*/ TransactionOutputScript,
+                            TransactionOutputWithAddress,
+                            TransactionOutputWithScriptData /*, TransactionOutputs*/};
```

Comment:
If `TransactionOutput` and `TransactionOutputs` are deleted, it is preferable to remove them from imports instead of commenting out.